.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "jbmc/lib/java-models-library"]
 	path = jbmc/lib/java-models-library
 	url = https://github.com/diffblue/java-models-library.git
+[submodule "lib/variant"]
```

Comment:
Instead of introducing a new top-level folder, can we place this in `nonstd/variant` instead of `lib/variant`?